### PR TITLE
Refactor file downloads

### PR DESCRIPTION
**Pull request by evansiroky**
This PR changes the way files are downloaded. The `gtfsAndOsmUrls` manifest entry is now replaced with `routerFolderDownloads`.  All files in the `routerFolderDownloads` are now downloaded to the router folder every time even if otp-runner is configured to run the server only. This has the advantage of being able to download extra files such as the build/router-config.json files, elevation files or whatever other file needs to be in the router folder.

Also, this PR better handles multi-line logs during graph building.

This PR depends on #2.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -179,10 +182,10 @@ module.exports = class OtpRunner {
       errors.push('At least one of `buildGraph` or `runServer` must be set to true')
     }
 
-    // if build is set to true, then gtfsAndOsmUrls needs to be defined
-    if (buildGraph && (!gtfsAndOsmUrls || !gtfsAndOsmUrls.length)) {
-      errors.push('`gtfsAndOsmUrls` must be populated for graph build')
-      console.log(gtfsAndOsmUrls)
+    // if build is set to true, then routerFolderDownloads needs to be defined
+    if (buildGraph && (!routerFolderDownloads || !routerFolderDownloads.length)) {
```
Comment (landonreed) [inline lib/index.js:186]: Would a routerFolderDownloads = 'some-string-value' be caught in validation before this line?

Comment (evansiroky) [reply lib/index.js:186]: Yes, there is a JSON Schema validation that occurs before this.

**Comment by evansiroky** [comment]
I changed the branch to dev.

**Comment by binh-dam-ibigroup** [comment]
Running `yarn test` added the following to `otp-runner.js.snap`, is that intended?
```js
exports[`otp-runner failures should fail if the graph build fails 3`] = `
Object {
  "error": true,
  "graphBuilt": false,
  "graphUploaded": false,
  "message": "Failed to download file: s3://mock-bucket/gtfs.zip. Error: Error: No mock found for execa args!",
  "nonce": "bad-jar-download-nonce",
  "numFilesDownloaded": 0,
  "pctProgress": 10,
  "serverStarted": false,
  "totalFilesToDownload": 2,
}
`;

exports[`otp-runner failures should fail if the graph build fails 4`] = `
Object {
  "error": true,
  "graphBuilt": false,
  "graphUploaded": false,
  "message": "Build graph failed! Please see logs.",
  "numFilesDownloaded": 2,
  "pctProgress": 30,
  "serverStarted": false,
  "totalFilesToDownload": 2,
}
`;

exports[`otp-runner failures should fail if the graph build fails 5`] = `"Mock building failed!"`;
```

**Comment by binh-dam-ibigroup** [comment]
I am not sure whether this pertains to this PR, these two snapshots fail for me:
- `otp-runner failures should fail if the graph build fails 1`
- `otp-runner failures should fail if the graph build fails 2`

**Comment by binh-dam-ibigroup** [comment]
Sorry, never mind the failing snapshots.

**Comment by evansiroky** [comment]
I verified with @binh-dam-ibigroup that no extra snapshots were being generated.

**Pull request merged**